doc/advanced-configuration/Services.md

```diff
@@ -1,8 +1,25 @@
-### Module Description
-This module significantly extends `mongooseimctl` script capabilities. New functionalities appear as additional commands. For detailed description, just run `mongooseimctl` script with no parameters.
+Some functionalities in MongooseIM are provided by "services".
+A service is similar to a module, only it is started only once with a global configuration, while a module is started for every virtual host and may have global or host-specific configuration.
```

Comment:
A service is similar to a module, but while a module is started for every virtual host and may have global or host-specific configuration, a service is started only once with global configuration.